tests/wolfssl_crypto_benchmark/Makefile

```diff
@@ -0,0 +1,48 @@
+APPLICATION = wolf_crypto_benchark
+
+# If no BOARD is found in the environment, use this default:
+#BOARD ?= native
+BOARD ?= samd21-xpro #Currently missing implementation for strncasecmp
+#BOARD ?= pic32-wifire
+#BOARD ?= frdm-k64f
+#BOARD ?= stm32f4discovery
```

Comment:
please remove the boards that are commented out. If you think its is important to note that it was tested on those boards you could just add a comment.